UXWColors.js

```diff
@@ -1,13 +1,344 @@
 function UXWColors(props) {
   return (
     <div className="container">
+      <br/>
+      <br/>
+      <p> - - !  Until this tag is removed, color swatches are not accurate to UXW colors ! - - </p>
+      <br/>
+      <br/>
+
+      <br/>
+      <div className="font-bold font-2xl">
       <p>
-        Lucas ipsum dolor sit amet carnor needa chazrach rebo sebulba plagueis finis di veknoid ventress.
-        Maul lando vader sy hutt nomi. Gordin haruun bail jar. Fett mothma toydarian jerec greedo bothawui sesswenna artaru zuckuss.
-        Nassau cerea sern naboo. Tagge leia obi-wan thennqora antilles. Davin selkath jobal tavion metalorn organa muunilinst jettster.
-        jinn hutt ikrit vor. Bertroff kael hutt wharl oola dagobah bibble saleucami. Colton raioballo ansionian hapan billaba wampa organa.
-        Greedo argazdan jinn tof verpine bothan wedge.
+        Philosophy
       </p>
+      </div>
+      <br/>
+
+      <p>
+      Our palette consists of neutral, pleasing tones balanced by vivid
+      and eccentric colors. This color palette has been carefully chosen
+      to provide versatility and function while still providing boundless
+      room for expression.
+      </p>
+
+      <br/>
+      <br/>
+      <br/>
+
+      <div className="font-bold font-2xl">

```

Comment:
**suggestion**: you can change this to something like 

```html
<h2 className="font-bold font-2xl">Using our palette</h2> 
```

You can add classes to **any** HTML element

Reply:
oh, maybe a silly question, but what is a div then? I thought that was like... the tag that means tell it to change something but I guess that's wrong?